index.md

```diff
@@ -57,59 +60,79 @@ mode.
+            <div class="component">
+                <div class="component-icon">
+                    <a href="docker-cloud/"> <img src="../images/cloud_48.svg" alt="Docker Cloud"> </a>
                 </div>
+                <h3 id="docker-cloud"><a href="docker-cloud/">Docker Cloud</a></h3>
+                <p>A hosted service for building, testing, and deploying Docker images to your hosts.</p>
             </div>
-            <!--UCP-->
-            <div class="component-full">
-                <div class="component-full-icon">
-                    <img src="../images/UCP_48.svg" alt="Docker Universal Control Plane">
+        </div>
+    <!--UCP-->
+        <div class="col-sm-4 col-md-12 col-lg-3 block">
+            <div class="component">
+                <div class="component-icon">
+                    <a href="datacenter/ucp/2.1/guides/"> <img src="../images/UCP_48.svg" alt="Docker Universal Control Plane"> </a>
                 </div>
-                <div class="component-full-copy">
-                    <h3 id="docker-cloud"><a href="datacenter/ucp/1.1/overview/">Docker Universal Control Plane</a></h3>
-                    <p>(UCP) Manage a cluster of on-premises Docker hosts as if they were a single machine.</p>
+                <h3 id="ucp"><a href="datacenter/ucp/2.1/guides/">Docker Universal Control Plane</a></h3>
+                <p>(UCP) Manage a cluster of on-premise Docker hosts like a single machine with this enterprise product.</p>
+            </div>
+        </div>
+    <!--row 3-->
+    <!--compose-->
+        <div class="col-sm-4 col-md-12 col-lg-3 block">
+            <div class="component">
+                <div class="component-icon">
+                    <a href="compose/overview/"> <img src="../images/compose_48.svg" alt="Docker Compose"> </a>
                 </div>
+                <h3 id="compose"><a href="compose/overview/">Docker Compose</a></h3>
+                <p>Define application stacks built using multiple containers, services, and swarm configurations.</p>
             </div>
-            <!--compose-->
-            <div class="component-full">
-                <div class="component-full-icon">
-                    <img src="../images/compose_48.svg" alt="Docker Compose">
+        </div>
+    <!--DTR-->
+        <div class="col-sm-4 col-md-12 col-lg-3 block">
+            <div class="component">
+                <div class="component-icon">
+                    <a href="datacenter/dtr/2.2/guides/"> <img src="../images/dtr_48.svg" alt="Docker Trusted Registry"> </a>
                 </div>
-                <div class="component-full-copy">
-                    <h3 id="docker-cloud"><a href="compose/overview/">Docker Compose</a></h3>
-                    <p>Define applications built using multiple containers.</p>
+                <h3 id="dtr"><a href="datacenter/dtr/2.2/guides/">Docker Trusted Registry</a></h3>
+                <p>(DTR) stores and signs your images.</p>
```

Comment:
On smaller screens, the DTR card is looking funny because this description is too short compared with the other ones. Can you updated it to be something like

"An enterprise-grade image storage solution that you can install behind your firewall
and control who has access to your images."